web/src/routes/MyPages/ProfilePage.tsx

```diff
@@ -1,5 +1,347 @@
+
+  // Input Data list
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = e.target;
+    switch (name) {
+      case 'Email':
+        dispatch({ type: 'SET_EMAIL', payload: value });
+        break;
+      case 'Password':
+        dispatch({ type: 'SET_PASSWORD', payload: value });
+        break;
+      case 'Password2':
+        dispatch({ type: 'SET_CONFIRM_PASSWORD', payload: value });
+        break;
+      case 'Name':
+        dispatch({ type: 'SET_NAME', payload: value });
+        break;
+      case 'PhNumber':
+        dispatch({ type: 'SET_PHONE_NUMBER', payload: value });
+        break;
+      case 'Key':
+        dispatch({ type: 'SET_AUTH_KEY', payload: value });
+        break;
+      default:
+        break;
+    }
+  };
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+  };
+
+  // Clicked Complete UserRegist Button. so We need to do the last check required for registration.
+  const handleClick = () => {
+    logConsole('이메일 인증 여부: ', state.Email_Auth);
+    logConsole('중복 비밀번호 확인 여부 : ', state.Password_Overlap);
+    logConsole('인증번호 요청여부 확인 : ', state.Key_Request);
+    logConsole('인증번호 인증여부 : ', state.Key_Auth);
+    logConsole('서비스 이용약관여부 : ', state.AgreeService);
+    logConsole('개인정보 수집동의 여부 : ', state.AgreeInfor);
+  };
+
+  return (
+    <Container theme={theme}>
+      <Header theme={theme} toggleTheme={toggleTheme} />
+      <MypageContainer theme={theme}>
+        <Form onSubmit={handleSubmit}>
+          <MyPageTitle theme={theme}>마이페이지</MyPageTitle>
+
+          <FontText theme={theme}>이메일</FontText>
+          <ContainerBox>
+            <Input
+              type="email"
+              name="Email"
+              value={state.Email}
+              onChange={handleInputChange}
+              theme={theme}
+              placeholder="이메일을 입력하세요."
+            />
+            <ChangeButton type="submit" theme={theme}>
+              인증요청
+            </ChangeButton>
+          </ContainerBox>
+
+          <FontText theme={theme}>비밀번호</FontText>
+          <Input
+            type="password"
+            name="Password"
+            value={state.Password}
+            onChange={handleInputChange}
+            theme={theme}
+            placeholder="비밀번호를 입력하세요."
+          />
+
+          <FontText theme={theme}>인증코드</FontText>
+          <ContainerBox>
+            <Input
+              type="tel"
+              name="Key"
+              value={state.Key}
+              theme={theme}
+              onChange={handleInputChange}
+              pattern="[0-9]{4}"
+              maxLength={4}
+              placeholder="인증코드 4 자리를 입력하세요.(1234)"
+            />
+            <ChangeButton type="submit" theme={theme}>
+              인증하기
+            </ChangeButton>
+          </ContainerBox>
+
+          <AgreeBox>
+            <UserAgree
+              type="checkbox"
+              name="AgreeServ"
```

Comment:
I've got surprised when I saw this `AgreeServ`. Please rename it with full name.